kolibri/plugins/coach/assets/src/views/create-exam-page/index.vue

```diff
@@ -25,21 +25,74 @@
       @input="validateNum = true"
     />
 
-    <div v-if="channelExercises">
-      <h2>Choose exercises</h2>
-      <textbox
-        :ariaLabel="$tr('searchContent')"
-        :placeholder="$tr('searchContent')"
-        v-model.trim="searchInput"
-      />
-      <div v-if="searchInput">
-        search results
-      </div>
-      <div v-else>
-        breadcrumbs
-        topic list
+    <h2>Choose exercises</h2>
+    <textbox
+      :ariaLabel="$tr('searchContent')"
+      :placeholder="$tr('searchContent')"
+      v-model.trim="searchInput"
+    />
+    <div v-if="searchInput">
+      search results
+    </div>
+    <div v-else>
+      <nav>
+        <ol>
+          <li v-for="(topic, index) in topic.breadcrumbs" :class="[notLastBreadcrumb(index) ? 'not-last' : '']">
+            <button v-if="notLastBreadcrumb(index)" @click="handleGoToTopic(topic.id)">{{ topic.title }}</button>
+            <strong v-else>{{ topic.title }}</strong>
+          </li>
+        </ol>
+      </nav>
+
+      <div>
+        <table>
+          <thead>
+            <tr>
+              <th class="col-icon"></th>
+              <th class="col-title">{{ $tr('title') }}</th>
+              <th class="col-add"></th>
+            </tr>
+          </thead>
+          <tbody v-if="!loading">
+            <topic-row
+              v-for="topic in subtopics"
+              :topicId="topic.id"
+              :topicTitle="topic.title"
+              @goToTopic="handleGoToTopic"
+              @addTopicExercises="handleAddTopicExercises"/>
+            <exercise-row
+              v-for="exercise in exercises"
+              :exerciseId="exercise.id"
+              :exerciseTitle="exercise.title"
+              @addExercise="handleAddExercise"/>
+            <p v-if="(subtopics.length === 0) && (exercises.length === 0)">No exercises within this topic.</p>
+          </tbody>
+          <tbody v-else>
+            LOADING...
+          </tbody>
+        </table>
       </div>
     </div>
+
+    <div class="footer">
+      <p>7 Exercises selected</p>
+      <icon-button :text="$tr('preview')" @click="preview">
+        <mat-svg category="action" name="visibility"/>
+      </icon-button>
+      <br>
+      <icon-button :text="$tr('finish')" :primary="true" @click="createExam(currentClass.id, currentChannel.id, exercisesSelected, seed)"/>
```

Comment:
It seems like this is the only instance you're using 'currentClass' and 'currentChannel'? Set as default values in the action and pull values from `store.state` so that you don't have unnecessary info in the view